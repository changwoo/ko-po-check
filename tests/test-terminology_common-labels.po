# -*- text -*-
# terminology/common-labels
#
msgid ""
msgstr ""
"Project-Id-Version: test 18\n"
"POT-Creation-Date: 2010-07-18 07:02+0900\n"
"PO-Revision-Date: 2010-07-18 07:02+0900\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

msgid "View Preview"
msgstr "미리보기 보이기"

msgid "View Preview..."
msgstr "미리보기 보기..."

# 예외
msgid "_View only"
msgstr "보기만 가능(_V)"
