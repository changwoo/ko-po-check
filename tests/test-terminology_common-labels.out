-:16: 미리보기 보이기: "View ..." 형태는 "... 보기" 형태와 같이 번역합니다
